docs/userGuide/syntax/headers.mbdf

```diff
@@ -23,6 +23,17 @@ In the page that you want to include the header:
 ```
 </div>
 
+You can fix the header to be always on the top by add `class="header-fixed"` attribute to the `<header>` tag of your header file:
+
+{{ icon_example }}
+**`_markbind/headers/`**`header.md`:
```

Comment:
Since we are documenting this now, would be a good point to get @damithc's opinion on fixed navbar usage.
( maybe something simpler like `<header fixed>`? 😮  )

Reply:
Yes I prefer a simpler `fixed` if there are no other drawbacks.